crates/anemo/src/crypto.rs

```diff
@@ -6,7 +6,9 @@ use crate::{error::AsStdError, PeerId};
 static SUPPORTED_SIG_ALGS: &[&webpki::SignatureAlgorithm] = &[&webpki::ED25519];
 
 #[derive(Clone, Debug)]
-pub(crate) struct CertVerifier(pub(crate) String);
+pub(crate) struct CertVerifier {
+    pub server_names: Vec<String>,
```

Comment:
similar question, can this be pub(crate) or why the bump in visibility?

Reply:
fixed